internal/client/config.go

```diff
@@ -148,7 +148,7 @@ func (c *Config) HasCredentials() bool {
 }
 
 func (c *Config) GetClientKeyPath() string {
-	return resolvePath(c.AuthInfo.ClientCertificate, c.baseDir)
+	return resolvePath(c.AuthInfo.ClientKey, c.baseDir)
```

Comment:
Change it as seems like it is a bug as there is GetClientCertificatePath for certificate path